music21/musicxml/xmlToM21.py

```diff
@@ -2995,6 +3014,42 @@ def xmlToPitch(self, mxNote, inputM21=None):
 
         return p
 
+    def xmlToUnpitched(self, mxUnpitched, inputM21=None) -> note.Unpitched:
+        '''
+        Set `displayStep` and `displayOctave` from `mxUnpitched`.
+
+        >>> from xml.etree.ElementTree import fromstring as EL
+        >>> MP = musicxml.xmlToM21.MeasureParser()
+        >>> MP.divisions = 10080
+
+        >>> mxNote = EL('<note><duration>7560</duration><type>eighth</type></note>')
+        >>> unpitched = EL('<unpitched>'
+        ...                + '<display-step>E</display-step>'
+        ...                + '<display-octave>5</display-octave>'
+        ...                + '</unpitched>')
+        >>> mxNote.append(unpitched)
+        >>> n = MP.xmlToSimpleNote(mxNote)
+        >>> n.displayStep
+        'E'
+        >>> n.displayOctave
+        5
+        >>> n.midi
+        76
+        '''
+        if inputM21 is None:
+            unp = note.Unpitched()
+        else:
+            unp = inputM21
+
+        mxDisplayStep = mxUnpitched.find('display-step')
+        mxDisplayOctave = mxUnpitched.find('display-octave')
+        if textStripValid(mxDisplayStep):
+            unp.displayStep = mxDisplayStep.text.strip()
+        if textStripValid(mxDisplayOctave):
+            unp.displayOctave = int(mxDisplayOctave.text.strip())
```

Comment:
I'd like to make a TODO to compensate for MusicXML number of staff-lines affects the display-step behavior, which I don't want to adopt in music21.  In a one line staff, B4 should equal center line in music21 (as it does in most other notation settings).  In MusicXML which line an Unpitched is displayed on depends on the number of stafflines currently active.

See https://github.com/w3c/musicxml/issues/351